plugins/database/src/general_query.cpp

```diff
@@ -165,12 +165,10 @@ sFklink( const char *table1, const char *table2, const char *connectingSQL ) {
     Links[nLinks].table1 = fkFindName( table1 );
     Links[nLinks].table2 = fkFindName( table2 );
     snprintf( Links[nLinks].connectingSQL, sizeof( Links[nLinks].connectingSQL ), "%s", connectingSQL );
-    if ( debug > 1 ) printf( "link %d is from %d to %d\n", nLinks,
-                                 Links[nLinks].table1,
-                                 Links[nLinks].table2 );
-    if ( debug2 ) printf( "T%2.2d L%2.2d T%2.2d\n",
-                              Links[nLinks].table1,  nLinks,
-                              Links[nLinks].table2 );
+    if (debug > 1)
+        log_gq::debug("{}: link {} is from {} to {}", __func__, nLinks, Links[nLinks].table1, Links[nLinks].table2);
```

Comment:
`{}` -> `[{}]`

- Skip the one for the function name
- Apply to all other locations you touched